examples/dockerdev/.dockerdev/Dockerfile

```diff
@@ -14,11 +14,17 @@ RUN apt-get update -qq \
     curl \
     less \
     git \
+    shared-mime-info \
   && apt-get clean \
   && rm -rf /var/cache/apt/archives/* \
   && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
   && truncate -s 0 /var/log/*log
 
+RUN cp /usr/share/mime/packages/freedesktop.org.xml ./ \
```

Comment:
This step is excess because of image size will not be reduced at the end.

Reply:
Yeah.

We should either move the first `cp` and the `apt-get remove` commands to the `RUN` above or add an independent `RUN apt-get update ..... apt-get clean ...` just for this functionality.

I prefer the second approach.

@bibendi @PhilCoggins Thoughts?

Reply:
I prefer to just install the package and don't do any actions after :)

By the way, is this fix needed after the recent fixes at active-storage's dependencies?

Reply:
> By the way, is this fix needed after the recent fixes at active-storage's dependencies?

Not sure, but I just found that my app doesn't work without it 😁